sjb/children.sh

```diff
@@ -4,21 +4,24 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-function children() {
-	local config=$1
-
-	for child_config in sjb/config/test_cases/*.yml sjb/config/common/test_cases/*.yml; do
-		if ! grep -q 'parent' "${child_config}"; then
-			continue
-		fi
+CONFIGS='sjb/config/test_cases/*.yml sjb/config/common/test_cases/*.yml'
+declare -A PARENTS
```

Comment:
This is not portable and won't work on latest MacOS AFAIR

Reply:
Who uses mac anyw…